README.md

```diff
@@ -19,12 +19,16 @@ locally, please checkout the [Plugin installation docs](https://grafana.com/docs
 
 ### ClickHouse user for the data source
 
-Set up an ClickHouse user account with `readonly` permission and access to
+Set up an ClickHouse user account with [readonly](https://clickhouse.com/docs/en/operations/settings/permissions-for-queries#settings_readonly) permission and access to
 databases and tables you want to query. Please note that Grafana does not
 validate that queries are safe. Queries can contain any SQL statement. For
 example, statements like `ALTER TABLE system.users DELETE WHERE name='sadUser'`
 and `DROP TABLE sadTable;` would be executed.
 
+To configure a readonly user, follow these steps:
+
+1. Ensure the readonly user has enough permission to modify the `max_execution_time` setting required by the underlying [clickhouse-go client](https://github.com/ClickHouse/clickhouse-go/).
```

Comment:
Is it obvious on how to configure the user with this specific permission? Or should we provide a  link to documentation on that?

Reply:
@asimpson added a link to the docs